src/app/containers/RadioPageBlocks/Blocks/OnDemandHeading/index.test.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { shouldMatchSnapshot } from '@bbc/psammead-test-helpers';
+import { ServiceContextProvider } from '#contexts/ServiceContext';
+import OnDemandHeading from '.';
+
+describe('AudioPlayer blocks OnDemandHeading', () => {
+  shouldMatchSnapshot(
```

Comment:
Snapshots on their own don't really give us much information about what the component should do. Can we also have tests that check for the role attribute and the hidden comma e.g. `should have role=text attribute so that screen readers read the headline in one go` and `should have comma with tabindex=-1 so screenreaders pause when reading` or something like that. Otherwise someone might not understand why it's there and then remove or change things

Reply:
Add to the a11y section of readme?

Reply:
Good idea @jroebu14 will add those tests

@greenc05 - which readme do you mean?

Reply:
Sorry maybe there isn't a readme like the psammead components have